### PR TITLE
Fix: Weird margins on the socials in Footer on landscape mode.

### LINKED ISSUE #91: Bug: Weird margins on the socials in Footer on landscape mode.
### Describe the bug

On certain mobile devices/layouts the socials are missing margins on the right: 

OpenSauced footer bug

### Steps to reproduce

1.) Go to developer tools
2.) Select mobile view
3.) Select "IPhone 12/13" on the devices drop-down menu.
4.) Switch to landscape mode
5.) Scroll down to Footer section.

At the top are my dev-tool settings as displayed in Firefox:
OpenSauced footer bug settings

I was able to replicate the issue in Google Chrome as well.

### Affected services

opensauced.pizza

### Platforms

Mobile

### Browsers

Chrome, Firefox

### Environment

Production

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by dev-phantom**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

Added  some margin right to the social icons
## Related Tickets & Documents

Fixes #91 
## Mobile & Desktop Screenshots/Recordings
image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?

no

### CODE REVIEW

File: lib/sanity.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ import {
 } from '../types/schema'
 
 const client = sanityClient({
-  projectId: process.env.SANITY_PROJECT_ID,
+  projectId: process.env.SANITY_PRO,
```
Comment (bdougie) [inline lib/sanity.ts:18]: What is this @dev-phantom?

**Comment by bdougie** [comment]
> Added some margin-right to the social icons

Thanks for opening the PR @dev-phantom, but I think there is a better solution—more of a band-aid. I commented on the issue where we should discuss the solution. Also, recommend adding more context to why you chose this solution.

**Review by bdougie — CHANGES_REQUESTED** [review]
Need a better solution.

File: lib/sanity.ts (hunk 1 above)
Comment (dev-phantom) [reply lib/sanity.ts]: sorry about that i have fixed it

File: components/sections/Footer.tsx [hunk 2]
```diff
@@ -49,7 +49,7 @@ const Footer: FC<FooterProps> = ({ pressPage }) => {
           </div>
         </div>
         <div className="w-full flex flex-col largeTablet:flex-row">
-          <div className="w-full largeTablet:w-[45%]">
+          <div className="w-full largeTablet:w-[45%] mr-5">
```
Comment (shamimbinnur) [inline components/sections/Footer.tsx:52]:
```suggestion
          <div className="w-full largeTablet:w-[45%]">
```

File: components/sections/Footer.tsx [hunk 3]
```diff
@@ -49,7 +49,7 @@ const Footer: FC<FooterProps> = ({ pressPage }) => {
           </div>
         </div>
         <div className="w-full flex flex-col largeTablet:flex-row">
```
Comment (shamimbinnur) [inline components/sections/Footer.tsx:51]:
```suggestion
        <div className="w-full flex flex-col largeTablet:flex-row flex-wrap gap-2">
```

**Review by shamimbinnur — COMMENTED** [review]
@dev-phantom  These changes should fix the margin and the responsiveness issue for all the elements on the footer.

**Comment by dev-phantom** [comment]
@shamimbinnur it seems the issue has been reassigned to someone else by @bdougie 
That was why I didn't do it

**Comment by dev-phantom** [comment]
@bdougie I have fixed this
please recheck

File: components/common/SocialLinks.tsx [hunk 4]
```diff
@@ -47,7 +47,7 @@ interface SocialLinksProps {
 const SocialLinks: FC<SocialLinksProps> = ({ aboutPage }): ReactElement => {
   const wrapperStyle = aboutPage
```
Comment (shamimbinnur) [inline components/common/SocialLinks.tsx:48]: ` const wrapperStyle = aboutPage
   ? 'grid grid-cols-3 gap-x-8 gap-y-3 largeTablet:flex'
   : 'flex flex-wrap relative gap-x-3 pr-2'
`

Why do we need a specific style only for the about page? What's the difference between the about page and all other pages?

Comment (dev-phantom) [reply components/common/SocialLinks.tsx:48]: Noticed that also but that was how I met it in the repo

**Review by bdougie — APPROVED** [review]
This works

**Comment by bdougie** [comment]
@dev-phantom sorry this has been sitting for a while. I could only review on mobile when I saw this opened. In the future, a change like this should also include a video or gif to make it faster to review.

**Comment by bdougie** [comment]
@dev-phantom would be able to create a highlight about your work on this quick fix?

https://opensauced.pizza/blog/social-coding-is-back

**Pull request merged**